package-lock.json

```diff
@@ -0,0 +1,8368 @@
+{
```

Comment:
Should this be checked in?

Reply:
yes, this is going to ensure everyone gets exactly the same dependency tree as me.